tests/test_callback_best_epoch.py

```diff
@@ -71,7 +78,7 @@ def lightning_classification_pipeline(
         LightningClassificationPipeline(
             output_path=result_path.name,
             **pipeline_kwargs,
-            **dataset_kwargs,
+            **dataset_kwargs[0],
```

Comment:
I'd consider adding comments for future us in couple of months

Reply:
Added comment but in the fixture function of that object.